ext/include/opentelemetry/ext/zpages/zpages.h

```diff
@@ -0,0 +1,66 @@
+#pragma once
+
+#include <chrono>
+#include <iostream>
+#include <memory>
+
+#include "opentelemetry/ext/zpages/tracez_data_aggregator.h"
+#include "opentelemetry/ext/zpages/tracez_http_server.h"
+#include "opentelemetry/ext/zpages/tracez_processor.h"
+
+#include "opentelemetry/nostd/shared_ptr.h"
+#include "opentelemetry/sdk/trace/tracer_provider.h"
+#include "opentelemetry/trace/provider.h"
+
+using opentelemetry::ext::zpages::TracezDataAggregator;
+using opentelemetry::ext::zpages::TracezHttpServer;
+using opentelemetry::ext::zpages::TracezSpanProcessor;
+using std::chrono::microseconds;
+
+/**
+ * Wrapper for zPages that initializes all the components required for zPages,
+ * and starts the HTTP server in the constructor and ends it in the destructor.
+ * The constructor and destructor for this object is private to prevent
+ * creation other than by calling the static function Initialize(). This follows the
+ * meyers singleton pattern and only a single instance of the class is allowed.
+ */
+class zPages
```

Comment:
nit: I'm not sure if `zPages` class name is in alignment with coding style guidelines mentioned in CONTRIBUTING.md ?

Reply:
I changed this to ZPages.